database/models/UsersInRooms.js

```diff
@@ -0,0 +1,27 @@
+const { DataTypes } = require("sequelize");
+const sequelize = require("../index");
+
+const UsersInRooms = sequelize.define(
+  "users_in_rooms",
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+      primaryKey: true,
+      autoIncrement: true,
+    },
+    room_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+    user_id: {
+      type: DataTypes.INTEGER,
+      allowNull: false,
+    },
+  },
+  {
+    timestamps: false,
+  }
```

Comment:
Read about [sequelize associations](https://sequelize.org/master/manual/assocs.html) and describe them